Entities/Vehicles/Common/VehicleCommon.as

```diff
@@ -413,12 +413,19 @@ bool Vehicle_AddLoadAmmoButton(CBlob@ this, CBlob@ caller)
 	{
 		// put in what is carried
 		CBlob@ carryObject = caller.getCarriedBlob();
-		if (carryObject !is null && !carryObject.isSnapToGrid())  // not spikes or door
+		if (carryObject !is null && (!carryObject.isSnapToGrid() || carryObject.getName() == "seed"))  // not spikes or door, allow seeds
 		{
 			CBitStream callerParams;
 			callerParams.write_u16(caller.getNetworkID());
 			callerParams.write_u16(carryObject.getNetworkID());
-			caller.CreateGenericButton("$" + carryObject.getName() + "$", getMagAttachmentPoint(this).offset, this, this.getCommandID("putin_mag"), getTranslatedString("Load {ITEM}").replace("{ITEM}", carryObject.getInventoryName()), callerParams);
+
+			string iconName = "$" + carryObject.getName() + "$";
+			if (carryObject.hasTag("use inventory icon"))
```

Comment:
We don't need to check by tag, this can be done with no tag with the same results
```suggestion
			if (GUI::hasIconName("$" + carryObject.getInventoryName() + "$"))
```